anaconda/pyanaconda/bootloader.py

```diff
@@ -1908,16 +1908,208 @@ def is_valid_stage2_device(self, device, linux=True, non_linux=False):
 
     write_config = BootLoader.write_config
 
-class MacEFIGRUB(EFIGRUB):
+class MacEFIGRUB(XenEFI):
+    """Special EFI handling for macOS HFS+ ESP partition.
+
+    Typical GRUB2 installations would execute the script
+    located at /usr/libexec/mactel-boot-setup which would
+    modify the HFS+ ESP files and bless the specified efi.
+
+    However, we are not using GRUB at this time which would
+    cause that script to exit earlier.
+
+    In this class, we will execute the relevant commands
+    to symlink the efi file in the /System directory as well
+    the cfg file.  Lastly, macOS requires the bootable efi
+    file to be blessed.
+    """
+
+    def __init__(self):
+        super(MacEFIGRUB, self).__init__()
+        self._mountpoint = "/boot/efi" # fixme: extract from writeBootLoader()
```

Comment:
I am holding off refactoring other parts of the file until 4.0 goes RTM.  Until then, `/boot/efi` is hardcoded throughout other parts.